tests/test_isrTaskLSST.py

```diff
@@ -476,6 +477,38 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrBrighterFatter(self):
+        """Test processing of a flat frame."""
+        mock_config = self.get_mock_config_clean()
+        mock_config.doAddSource = True
+        mock_config.sourceFlux = [75000.0]
+        mock_config.isTrimmed = False
+        mock_config.doAddBrighterFatter = True
```

Comment:
Are we not putting sky, flat, everything else?  I think we want the whole thing (or a second test that does the whole thing).

Reply:
We already have the skyImage test which tests all of those. But I realized that to properly test the BF component, we need a separate test, so I only draw a source onto an blank image.

Reply:
The skyImage test does not (currently) have BF turned on for either the mock or the ISR.  I would like to see a test that has all of everything, even if we can't do pixel-perfect comparisons.